pkg/minishift/openshift/service.go

```diff
@@ -238,6 +220,60 @@ func filterAndUpdateServiceSpecs(routeSpec map[string][]ServiceWeight, namespace
 	return serviceSpecs
 }
 
+func serviceHasNodePort(serviceNodePort map[string]string, name string) bool {
```

Comment:
try to use `is` or `has` as naming to start with

Reply:
what about `doesServiceHaveNodePort` ?

Reply:
conventions are to use `is` or `has`

Reply:
Done.. `hasNodePort` :-)